core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -644,6 +621,37 @@ public synchronized ClientFileInfo getFileStatus(int fileId, boolean useCachedMe
   }
 
   /**
+   * Create a user local temporary folder in some StorageDir
+   * 
+   * @param the id of the block
+   * @param the initial bytes allocated for the block file
+   * @return the temporary path of the block file
+   * @throws IOException
+   */
+  public synchronized String getLocalBlockLocation(long blockId, long initialBytes)
```

Comment:
This is for temporary block location, right? How about renaming it to be "getLocalTemporaryBlockLocation"